changelog.txt

```diff
@@ -7,6 +7,7 @@ vNext
 - Added support for signing and verifying arbitrary String data with select RSA algorithms.
 - Added support for multiple software/hardware backed RSA keys using AsymmetricKey, AsymmetricKeyFactory.
 - Bugfix: Added a workaround for keypair generation on API < 23 devices using locales with non-Gregorian calendars. (#1075)
+- Enforce changelog update
```

Comment:
Does this actually need to be in the changelog?

Reply:
Mainly it was an illustration that it works.  I can remove it.